src/content/blog/astro-350.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: "Astro 3.5: i18n Routing"
+description: "Astro 3.5 is out! It includes experimental support for i18n routing, prefetch for speedier page loads, form support with ViewTransitions, image optimization improvements, and more."
+publishDate: "November 8, 2023"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-350/post-header-3.5.webp"
+socialImage: "/src/content/blog/_images/astro-340/og-image-3.4.webp"
```

Comment:
Social image is still pointing to the old 3.4.0 asset

Reply:
Yeah, I just added that.